cdap-docs/included-applications/source/etl/plugins/transforms/script.rst

```diff
@@ -72,6 +76,15 @@ schema is the same as the input schema.
           {\"name\":\"tax\",\"type\":\"double\"},
           {\"name\":\"total\",\"type\":\"double\"}
         ]
+      }",
+      "lookup": "{
+        \"purchases\":{
+          \"type\":\"DATASET\",
+          \"properties\":{
+            \"dataset_argument1\":\"foo\",
+            \"dataset_argument2\":\"bar\"
```

Comment:
Andreas has added some new material to this page (http://docs.cask.co/cdap/3.3.0-SNAPSHOT/en/developers-manual/building-blocks/datasets/index.html#using-datasets-in-programs), perhaps this can be an elaboration of these points. The datasets material could use some love in the form of fixing of the sidebar Table of Contents, for sure..